### ISSUE TITLE
setup the sia mqtt broker

### ISSUE THREAD

**Issue by inuites** [dklemm/FlexSIA2MQTT#1]
Hi dklemm, 

I own a FX100 panel and I would like to pick up status of mag contact and PIR sensors like you mentionned.
 
Can you tell me briefly how to run the FlewSIA2MQTT ? Do I just need to edit the config.yml and  run it directly in docker ?

Thank you very much for your help.

inuites.

**Comment by dklemm** [comment]
Yes, exactly as you describe.   although config.yml is linked using docker volume so you don't need all the code/repo just a copy of config.yml.  You need to configure you FX100 to output ARC to tcpip and set your zones to log function.  The mqtt client has no security setup, I should probably sort this.

**Comment by mvjt** [comment]
Hello @dklemm!

Awesome work. I´ve been looking for something like this for quite some time. I do have a problem getting this to run though. When I´ve built the container locally and specified the config.yml, starting the server just outputs the version and then exits?
The only output I get is v10.16.3

**Comment by Ruaktom** [comment]
Hi @dklemm ,

I’m also trying to set this up for my alarm system for use in Home Assistant. Am I misunderstanding or should I program my sensors to the log function for the code to work correctly? I would not get an MQTT message for zones programmed as intruder for example? 

Thanks!

Tom

**Comment by dklemm** [comment]
@Ruaktom sadly yes this will only work for zones set to "LOG" function.  I'm not an expert in Flex but I think you can create up-to 2 user defined functions they can be setup to LOG and ALARM.  This project can be updated to process the ALARM SIA codes.  See the handlers.
